Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -271,6 +271,13 @@ public void RefreshProjects(IVBE vbe)
             }
         }
 
+        private List<SyntaxErrorEventArgs> _errors = new List<SyntaxErrorEventArgs>();
+        internal void AddParserError(SyntaxErrorEventArgs e)
+        {
+            _errors.Add(e);
+            SetStatusAndFireStateChanged(this, ParserState.Error);
```

Comment:
If you can get multiple errors from the same parse, using this method might not be too great: it fires the state changed event every time it is called.